### DISCUSSION TITLE
Sankey diagram explanation

### DISCUSSION THREAD

**Discussion by Hd1004** [LCA-ActivityBrowser/activity-browser#790] [General]
![Screenshot (336)](https://user-images.githubusercontent.com/105209945/171012023-cba63d92-5f36-487d-b97a-5788b10077c5.png)
![Screenshot (335)](https://user-images.githubusercontent.com/105209945/171012028-16849a22-b9d3-401b-ae38-0afcaa1757e5.png)

Hey guys, can someone please explain to me these Sankey diagrams? What does the % in the boxes mean as opposed to the % beside the connecting lines? Also, in both diagrams, it shows that 50 and 56% impact is during the production process but is there a way to figure out which point of the production process has these impacts? Where the emissions are coming from specifically during the production process (production box with 50 and 56% impact)

**Comment by marc-vdm** [comment]
I'm not an expert on this, but as far as I understand, the processes and flows should be considered separately. 

e.g. in your lower figure, the `ammonia production` process contributes 50% of the impact. Another view is from the flows, where your could assume `heat, district or industrial, natural gas` produces 27.5% of the impact, considering all upstream processes (including the process `heat production, natural gas, at boiler...`. 

Note here, that the Sankey is based on graph traversal, it does not always represent the total impacts, just the largest ones. You can adjust how much is calculated with the `cutoff` and `Calculation depth` values. See also the [Brightway documentation](https://2.docs.brightway.dev/technical/bw2calc.html?highlight=graph#graph-traversal) for more info.
